modules/go-add-infra-nodes.adoc

```diff
@@ -4,7 +4,21 @@
 
 :_content-type: PROCEDURE
 [id="add-infra-nodes_{context}"]
-= Adding infrastructure nodes
+= Moving GitOps workloads to infrastructure nodes
+
```

Comment:
As suggested earlier, every module must contain an abstract that summarizes its content in a brief user-centered statement.